netsim/modules/bgp.yml

```diff
@@ -29,11 +29,11 @@ attributes:
     ebgp_role: str
     as_list: dict
     sessions:
-      ipv4: [ ibgp, ebgp ]
-      ipv6: [ ibgp, ebgp ]
+      ipv4: [ ibgp, ebgp, localas_ibgp ]
+      ipv6: [ ibgp, ebgp, localas_ibgp ]
     activate:
-      ipv4: [ ibgp, ebgp ]
-      ipv6: [ ibgp, ebgp ]
+      ipv4: [ ibgp, ebgp, localas_ibgp ]
```

Comment:
Most of the templates treat ibgp and localas_ibgp sessions the same way. What is the justification for them being different?

Reply:
We could decide to remove the ```localas_ibgp``` enum value and only support ```[ ibgp, ebgp ]```, but at the moment bgp neighbors of type ```localas_ibgp``` are being generated when ```bgp.local_as``` is set equal to the neighbor AS. The default ```activate``` and ```sessions``` values - if not set in a lab - include such 'localas_ibgp' sessions due to the Python string constant, but if I set the global sessions to some value there is no way to get such localas_ibgp sessions activated. They disappear silently

The justification is based on being able to configure all valid values, with consistency between the YAML files and what's defined by the Python source that accompanies them